src/test/java/pl/ergohestia/ehj1/ivesta/repository/RoutesLoaderTest.java

```diff
@@ -0,0 +1,27 @@
+package pl.ergohestia.ehj1.ivesta.repository;
+
+import org.junit.jupiter.api.Test;
+import pl.ergohestia.ehj1.ivesta.model.Route;
+
+import java.nio.file.Path;
+import java.util.List;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+class RoutesLoaderTest {
+
+    private RoutesLoader sut = new RoutesLoader();
+    private static final Path TEST_PATH = Path.of(("src/test/resources/testRoutes.csv"));
+
+    @Test
+    void shouldGetListOfRoutesFromCSV() {
```

Comment:
ten test jest do wywalenia, bo nie mamy tras wczytywanych z pliku

Reply:
to już jest wyrzucone